docs/source/user_guide/runtime-image-conf.md

```diff
@@ -118,6 +118,24 @@ $ elyra-metadata install runtime-images --schema_name=runtime-image \
        --replace
 ```
 
+#### Exporting runtime image configurations
+
+To export runtime image configurations:
+
+```bash
+elyra-metadata export runtime-images \
+	--schema-name="runtime-image" \
```

Comment:
```suggestion
	--schema_name="runtime-image" \
```

Reply:
Since there's only one schemaspace in the `runtime-images` namespace, we should not display the `--schema_name="runtime-image"`  option in the example and the explanatory text below.

Reply:
Removed all mentions of `--schema-name` parameter from the documentation as discussed via chat.